beets/util/artresizer.py

```diff
@@ -172,7 +182,7 @@ def resize(self, maxwidth, path_in, path_out=None, quality=0,
 
         return path_out
 
-    def get_size(self, path_in):
+    def get_size(self, path_in: str) -> Optional[Tuple[int, ...]]:
```

Comment:
```suggestion
    def get_size(self, path_in: bytes) -> Optional[Tuple[int, int]]:
```

Note that `path_in` is passed through `syspath`, i.e. it is a path in beets-internal bytestring-representation.

The output type should match the PIL backend. Probably, we should eventually make the code here validate the IM result completely, including the being exactly length 2 (just catching the `IndexError` is definitely not enough (if such an error can even happen?)).